Umbraco-Heartcore/API-Documentation/Content-Management/forms/index.md

```diff
@@ -0,0 +1,305 @@
+# Content Management API for Umbraco Forms
+
+**BASE URL**: `https://api.umbraco.io`
+
+**API version** 2.1
+
+:::note
+Umbraco Forms is not available in all plans, see the [Pricing & Features](https://umbraco.com/umbraco-heartcore-pricing/) for an overview of which plans it's included in.
```

Comment:
Suggested change:
The availability of Umbraco Forms depend on the plan, see the [Pricing & Features](https://umbraco.com/umbraco-heartcore-pricing/) for an overview of which plans includes Forms.

Reply:
Changed